tests/test_completeness.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+from pytest import raises
+
+from splink.exceptions import SplinkException
+from splink.exploratory import completeness_chart
+from tests.decorator import mark_with_dialects_excluding
+
+
+# The UNION ALL used for this chart gives a
```

Comment:
Note this was a bug before, there just wasn't a sqlite test of the completeness chart, so it wasn't apparent